rustls/src/conn.rs

```diff
@@ -1378,7 +1372,7 @@ pub(crate) struct Quic {
     /// QUIC transport parameters received from the peer during the handshake
     pub(crate) params: Option<Vec<u8>>,
     pub(crate) alert: Option<AlertDescription>,
-    pub(crate) hs_queue: VecDeque<(bool, Vec<u8>)>,
+    pub(crate) hs_queue: std::collections::VecDeque<(bool, Vec<u8>)>,
```

Comment:
Style nit: I'm not sure why you changed the import style here? This way seems unidiomatic.

Reply:
This struct has `#[cfg(feature = "quic")]`, so it is only parsed when that feature is enabled. And this is now the only use of VecDeque in this file.

If I `use std::collections::VecDeque` at the top of the file, I get clippy errors (unused use) for builds that don't include quic. There's currently no smaller scope with `#[cfg(feature = "quic")]` where I could put that `use`. I could refactor the quic-related code here to put it inside its own feature-gated `mod`, but that seemed a little beyond the scope of the refactoring at hand.

Reply:
We usually just guard the import itself with `#[cfg(feature = "quic")]` IIRC.